engine/src/main/java/org/terasology/logic/inventory/ItemComponent.java

```diff
@@ -80,6 +80,10 @@
      */
     public int baseDamage = 1;
 
+    public boolean pickableUp = true;
```

Comment:
IIRC "Pickable" actually has been a valid technical term for that sort of object :-)

If this is specific to delayed, uh, picks? I guess it could be DelayedPickableComponent. Or if it entirely splits out stuff related to picking up items then plain old PickableComponent sounds fine to me. @Josharias is the expert so get his opinion too :-)